checkmates/pipelines/transformers.py

```diff
@@ -83,6 +84,51 @@ def _get_feature_provenance(self):
         return {}
 
 
+"""Component that normalizes skewed distributions using the Yeo-Johnson method"""
+
+
+class SimpleNormalizer(Transformer):
```

Comment:
let's also add some basic tests for the normalizer to ensure it works as expected!

Reply:
added